ForumCrawler/ForumCrawler.csproj

```diff
@@ -558,8 +555,6 @@
     <EmbeddedResource Include="Migrations\202005091205228_RemoveExemptAndShowScore.resx">
       <DependentUpon>202005091205228_RemoveExemptAndShowScore.cs</DependentUpon>
     </EmbeddedResource>
-    <EmbeddedResource Include="Migrations\202005102158383_InertiaRevamp.resx">
-      <DependentUpon>202005102158383_InertiaRevamp.cs</DependentUpon>
```

Comment:
Migrations are purely for tracking the history of the database. What you're doing is the equivalent of removing a specific commit in the timeline of a git repository - if your aim is to remove inertia from the database, create a new migration which drops the entry.